packages/mermaid/src/docs/intro/getting-started.md

```diff
@@ -1,27 +1,62 @@
-# A Mermaid User-Guide for Beginners
+---
+outline: 'deep' # shows all h3 headings in outline in Vitepress
+---
+
+# Mermaid User Guide
+
+## Mermaid is composed of three parts
+
+1. Deployment
+1. Syntax
+1. Configuration
+
+This section talks about the different ways to **deploy** Mermaid.
+
+If you are a beginner:
+
+- Check out the [Diagram Syntax](syntax-reference.md) page
+- Check out the [Tutorials](../config/Tutorials.md) page
+
+## Ways to use Mermaid
+
+1. [Using the Mermaid Chart Editor](getting-started.html#_1-using-the-mermaid-chart-editor)
+1. [Using the Mermaid Live Editor](getting-started.html#_1-using-the-mermaid-chart-editor)
+1. [Using Mermaid Plugins and Integrations](getting-started.html#_3-using-mermaid-plugins)
+1. [Calling the Mermaid JavaScript API](getting-started.html#_4-calling-the-mermaid-javascript-api)
+1. [Adding Mermaid as a dependency](getting-started.html#_5-adding-mermaid-as-a-dependency)
```

Comment:
We should prefer linking to `.md` instead of `.html`, so that the docs still work in markdown, but because these links are in the same file, we don't need them at all!

Also, the live editor link is pointing to the wrong place.

I've also updated the numbering, since although Markdown will automatically output the correct numbers, there are some users that will read the raw text, so we should the correct numbers there too.

```suggestion
1. [Using the Mermaid Chart Editor](#_1-using-the-mermaid-chart-editor)
2. [Using the Mermaid Live Editor](#_2-using-the-mermaid-live-editor)
3. [Using Mermaid Plugins and Integrations](#_3-using-mermaid-plugins)
4. [Calling the Mermaid JavaScript API](#_4-calling-the-mermaid-javascript-api)
5. [Adding Mermaid as a dependency](#_5-adding-mermaid-as-a-dependency)
```